lib/wsdl.js

```diff
@@ -344,6 +344,19 @@ DefinitionsElement.prototype.init = function() {
 DocumentationElement.prototype.init = function() {
 };
 
+SchemaElement.prototype.merge = function(source) {
+  assert(source instanceof SchemaElement);
+  if (this.$targetNamespace === source.$targetNamespace) {
+    _.merge(this.complexTypes, source.complexTypes);
+    _.merge(this.types, source.types);
+    _.merge(this.elements, source.elements);
+    _.merge(this.xmlns, source.xmlns);
+    this.includes.concat(source.includes);
```

Comment:
Are you shure you didn't want to write `this.includes = this.includes.concat(source.includes);`? As `Array.prototype.concat` only returns a new concatenated Array but leaves the original Arrays untouched, `this.includes` would not change and therefore not contain the contents of `source.includes`.